web/app.ts

```diff
@@ -66,6 +67,25 @@ function startRecording(microphone: MediaStream) {
 
 async function onRecordDataAvailable(blob: Blob) {
   const base64Data = (await blobToBase64(blob)) as string;
+  dataToSend.push(base64Data);
+}
+
+console.log("init");
```

Comment:
Let's remove or:
```suggestion
console.log("Initialize XL8 Skroll E2E client...");
```
?